file_aloc/__init__.py

```
@@ -1,32 +1,263 @@
+
+    files: CollectionProperty(
+        name="File Path",
+        type=bpy.types.OperatorFileListElement,
+    )
+
+    aloc_filepath: StringProperty(
+        name="Collision Path",
+        description="Path to the Collision (ALOC) file",
+    )
+
+    def draw(self, context):
```

Ruff (0.14.8)

Unused method argument: `context`

(ARG002)